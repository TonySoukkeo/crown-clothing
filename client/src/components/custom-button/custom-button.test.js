import React from "react";
import { shallow, configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

import CustomButton from "./custom-button.component";

configure({
  adapter: new Adapter(),
});

describe("CustomButton component", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<CustomButton />);
  });

  it("renders CustomButton without crashing", () => {
    expect.assertions(1);
    expect(wrapper).toMatchSnapshot();
  });
});
