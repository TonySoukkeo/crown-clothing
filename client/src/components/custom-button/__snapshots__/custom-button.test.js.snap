// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomButton component renders CustomButton without crashing 1`] = `ShallowWrapper {}`;
